influxdb/server.sls

```diff
@@ -77,6 +77,22 @@ influxdb_create_db_{{db.name}}:
       - cmd: influxdb_create_admin
     {%- endif %}
   # TODO: manage database deletion
+
+    {% for rp in db.get('retention_policy', []) %}
+    {% set rp_name = rp.get('name', 'autogen') %}
+    {% set duration = rp.get('duration', 'INF') %}
+    {% set replication = rp.get('replication', '1') %}
+    {% set shard_duration = rp.get('shard_duration', False) %}
+    {% set query_rentention_policy = 'RETENTION POLICY {} ON {} DURATION {} REPLICATION {}'.format(
+        rp_name, db.name, duration, replication)
+    %}
+influxdb_retention_policy_{{db.name}}{{ rp_name }}:
```

Comment:
Do you want to add an underscore between the db name and the rp name in the declaration ID?